var/spack/repos/builtin/packages/py-pennylane/package.py

```diff
@@ -10,25 +10,45 @@
 class PyPennylane(PythonPackage):
     """PennyLane is a Python quantum machine learning library by Xanadu Inc."""
 
-    homepage = "https://github.com/XanaduAI/pennylane"
+    homepage = "https://docs.pennylane.ai/"
+    git = "https://github.com/PennyLaneAI/pennylane.git"
     pypi = "PennyLane/PennyLane-0.28.0.tar.gz"
 
-    maintainers("marcodelapierre")
+    maintainers = ["mlxd", "AmintorDusko"]
```

Comment:
```suggestion
    maintainers("mlxd", "AmintorDusko")
```
@marcodelapierre do you still want to be listed as a maintainer?

Reply:
yes, if you guys don't mind

Reply:
Happy for @marcodelapierre to remainon he list

Reply:
Hi @marcodelapierre, I'm happy to have you as a maintainer. 
Thank you for that!